docs/source/brain.rst

```diff
@@ -16,6 +16,15 @@ Board
 Microcontroller
 ===============
 
+The Bluepill board microcontroller reference is STM32F103CBT6 LQFP64,
+that means:
+
+* [STM32] ST microcontroller ARM Cortex M3-based with 32-bit 
+
+* [03] XL-density device with Flash memory density between 768 Kbytes and 1 Mbyte 
+
+* [LQFP64] Pinout
+
```

Comment:
I would rather say "The microcontroller reference is..." (this section is about the micro, not the board). There is an empty section just above in case you want to mention the Bluepill. :joy:

If you are using lists, then avoid blank lines between list items.

I would add the `F1` part, which is missing. And the `CBT6` too.

You may want to use definition lists here:

```
Item
  Definition

Another item
  Another definition

[...]
```

Reply:
I can not find the meaning of T6. Updated the rest of suggestions.